StatBot/Database/PostgresRepositories/PostgresStatsRepository.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace StatBot.Database.PostgresRepositories
+{
+    public class PostgresStatsRepository : IRepository<Statistic>
+    {
+        private StatDbContext _context;
+
+        public PostgresStatsRepository()
+        {
+            _context = new StatDbContext();
+        }
+
+        public void Add(Statistic stat)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename parameter 'stat' to 'o' to match the interface declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS927)